submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/SubmissionApiTest.kt

```diff
@@ -103,5 +115,28 @@ internal class SubmissionApiTest(tempFolder: TemporaryFolder) : BaseIntegrationT
             assertThat(savedSubmission).isNotNull
             assertThat(savedSubmission).isEqualTo(submission)
         }
+
+        @Test
+        fun `get projects`() {
```

Comment:
This test is supposed to contain submission related operations. I think this should be in a separated test class.